src/screens/ChangePasswordScreen.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import ProfileSection from '../components/ProfileSection';
+import { Text, StyleSheet, View, TextInput, ScrollView } from 'react-native';
+import BaseStyles from '../BaseStyles';
+import ButtonWithIcon from '../components/ButtonWithIcon';
+import PropTypes from 'prop-types';
+import AccountService from '../services/AccountService';
+import ChangePasswordModel from '../models/ChangePasswordModel';
+import { useDispatch } from 'react-redux';
+import { showSpinner, hideSpinner } from '../actions/application';
+import MessageModal from '../components/MessageModal';
+
+ChangePasswordScreen.propTypes = {
+	route: PropTypes.object
+};
+
+export default function ChangePasswordScreen() {
+	const [errorMessage, setErrorMessage] = useState(null);
+	const [valueCurrent, onChangeCurrentText] = React.useState('');
+	const [valueNew, onChangeNewText] = React.useState('');
+	const [valueRepeat, onChangeRepeatText] = React.useState('');
+
+	const dispatch = useDispatch();
+
+	const onSubmitTapped = async () => {
+		if(valueNew.length === 0 || valueRepeat.length === 0 || valueCurrent.length === 0) {
+			setErrorMessage('Por favor completa los datos para poder ingresar');
+			return;
+		}
+		if (valueNew !== valueRepeat) {
+			setErrorMessage('Las contraseñas no coinciden');
+			return;
+		}
+
+		dispatch(showSpinner);
+		try {
+			await AccountService.changePassword(new ChangePasswordModel(valueCurrent, valueNew));
+			setErrorMessage('Contraseña cambiada con exito');
+		}
+		catch (error) {
+			setErrorMessage(error.message);
```

Comment:
Please check if the error is instance of UserError before show the message. Check how this is handled in the login screen